sharepoint/sharepoint-ps/sharepoint-pnp/Connect-PnPOnline.md

```diff
@@ -466,6 +466,14 @@ Connect-PnPOnline -ClientId <id> -Certificate $cert -CertificatePassword (Conver
 
 Connects using an Azure Active Directory registered application using a certificate instance containing a private key. See https://docs.microsoft.com/en-us/sharepoint/dev/solution-guidance/security-apponly-azuread for a sample on how to get started.
 
+### ------------------EXAMPLE 25------------------
+```powershell
+Connect-PnPOnline -Url https://contoso.sharepoint.com -ClientId '<id>' -Tenant 'contoso.onmicrosoft.com' -Thumbprint 34CFAA860E5FB8C44335A38A097C1E41EEA206AA -TenantAdminUrl $tenantAdminUrl
```

Comment:
By passing the $tenantAdminUrl, we can perform admin activities like Add-PnPSiteCollectionAdmin. Otherwise, the commands will fail with insufficient permissions.